api/openapi.yaml

```diff
@@ -1626,6 +1628,46 @@ components:
       #  - asset
       #  - scan
 
+    AssetScanBenchmarks:
+      type: object
+      properties:
+        general:
+          $ref: '#/components/schemas/AssetScanBenchmark'
+        sbom:
+          $ref: '#/components/schemas/AssetScanBenchmark'
+        vulnerabilities:
+          $ref: '#/components/schemas/AssetScanBenchmark'
+        malware:
+          $ref: '#/components/schemas/AssetScanBenchmark'
+        rootkits:
+          $ref: '#/components/schemas/AssetScanBenchmark'
+        secrets:
+          $ref: '#/components/schemas/AssetScanBenchmark'
+        misconfigurations:
+          $ref: '#/components/schemas/AssetScanBenchmark'
+        exploits:
+          $ref: '#/components/schemas/AssetScanBenchmark'
+
+    AssetScanBenchmark:
+      type: object
+      properties:
+        scannedSize:
```

Comment:
Maybe it's better to put a list of inputs here that for each we'll state:
input type
Input path
Size
start/end time
number of files (optional)